src/components/Typography.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FunctionComponent, ReactNode } from "react";
+
+interface AProps {
+  href: string;
+  children: ReactNode;
+}
+
+export const A: FunctionComponent<AProps> = ({
+  href,
+  children,
+}: AProps): JSX.Element => (
+  <a href={href} className="govuk-link">
+    {children}
+  </a>
+);
```

Comment:
https://design-system.service.gov.uk/styles/typography/#links

Reply:
I agree, I had to stop myself from moving components in to Typography in this PR! I won't action now but worth looking at when we're mobbing today?

Reply:
Yeah agreed, just give a headsup to the rest of the team.  As the refactor will cover off pages already created as well as ones in progress